filament/backend/src/vulkan/platform/VulkanPlatform.cpp

```diff
@@ -673,12 +675,13 @@ Driver* VulkanPlatform::createDriver(void* sharedContext,
         mImpl->mDevice = scontext->logicalDevice;
         mImpl->mGraphicsQueueFamilyIndex = scontext->graphicsQueueFamilyIndex;
         mImpl->mGraphicsQueueIndex = scontext->graphicsQueueIndex;
+        context.mDebugUtilsSupported = scontext->debugUtilsSupported;
+        context.mDebugMarkersSupported = scontext->debugMarkersSupported;
+        context.mMultiviewEnabled = scontext->multiviewSupported;
```

Comment:
Can you move these lines next to the `!mImpl->mSharedContext` case?

Reply:
Below the declaration of ExtensionSet instExts? I kept it in the if (sharedContext){} block since that's the only place where VulkanSharedContext is

Reply:
I mean moving this to around 834 so that the two ways of setting these booleans are side-by-side.

Reply:
Ah ok I understand now